pom.xml

```diff
@@ -66,6 +66,12 @@
             <version>1.0.4</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
+            <version>2.6.0</version>
+            <optional>true</optional>
+        </dependency>
```

Comment:
Will this force my jenkins instance to install the branch-api plugin when I don't need it?

Reply:
It is marked as optional as stated here: https://www.jenkins.io/doc/developer/plugin-development/optional-dependencies/

So you do not need this dependency